plugins/team.ts

```diff
@@ -0,0 +1,248 @@
+import type { Plugin } from 'vite'
+
+export default function TeamItemFix(): Plugin {
+  return {
+    name: 'vitepress-team-member-item-fix',
+    enforce: 'pre',
+    async transform(code, id) {
+      if (id.includes('VPTeamMembersItem.vue') && !id.endsWith('.css') && !id.includes('&setup=')) {
+        return `
```

Comment:
That or just write a simple component yourself based on that and use it instead of vitepress' version. Alternatively, you can keep this as separate component and use aliases?

Reply:
removed vite plugins and included VP components + resolve entries

Reply:
Still curious, do we have any case where the avatar of one member is going to be larger than others? That's the only change I see from the vitepress' component here 👀

Reply:
I want to set w/h in the img tag, that's the change included

Reply:
Yeah but why is that needed? It’s immediate parent already has fixed height and width 👀